src/interfaces/store-interfaces/store-attribute-get-interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface StoreAttributeGetIF {
+  id: string,
+  path: string,
+
```

Comment:
Extra blank line.

Also, trailing commas are not needed (also in other interface below).

Reply:
I'll delete this file rather than fixing it.